### ISSUE TITLE
`ModuleNotFoundError` after using `micropip.install` on custom wheel url

### ISSUE THREAD

**Issue by hoodmane** [pyodide/pyodide#1683]
I'm facing a similar issue of running 

```py
import micropip
await micropip.install('./wheels/pyahocorasick-1.4.2-cp38-cp38-linux_x86_64.whl')
import ahocorasick
```

where `/wheels/pyahocorasick-1.4.2-cp38-cp38-linux_x86_64.whl` is a manually compiled wheel of `pyahocorasick` using `pip wheel`, but I'm getting `ModuleNotFoundError`. Any ideas on fixing this?

_Originally posted by @kevinlu1248 in https://github.com/pyodide/pyodide/issues/500#issuecomment-871803889_

**Comment by NishiPhalke** [comment]
Hi @hoodmane,

**Comment by NishiPhalke** [comment]
Hi @hoodmane,

**Comment by NishiPhalke** [comment]
Hi @hoodmane ,

I am trying install whl using micropip install URL(https://storage.googleapis.com/gcp.wenglab.org/bitarray-2.3.4-cp38-cp38-linux_x86_64.whl) created out of bitarray (https://github.com/ilanschnell/bitarray). But I am getting  this error when try to import bitarray.
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/lib/python3.9/site-packages/bitarray/__init__.py", line 14, in <module>
    from bitarray._bitarray import (bitarray, decodetree, _sysinfo,
ModuleNotFoundError: No module named 'bitarray._bitarray'

**Comment by NishiPhalke** [comment]
@rth Thanks for your detailed response!

**Comment by ryanking13** [comment]
This issue is resolved by #1859. Closing. If this behavior remains, please reopen this issue or make a new one.
